hs/MakeElements.hs

```diff
@@ -53,6 +53,13 @@ aboutLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.
                                        "static/style/graph/styles.css",
                                        "static/style/common/about.css",
                                        "static/style/common/common.css"])
+drawLinks :: H.Html
+drawLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css",
+                                       "static/style/graph/styles.css",
+                                       "static/style/graph/graph_styles.css",
+                                       "static/style/common/about.css",
+                                       "static/style/common/common.css"])
```

Comment:
Yes, I definitely don't need about.css, and is it just me or styles.css does not exist?